### PR TITLE
Integrate JSON Resume Schema : Work section

### PR DESCRIPTION
**Pull request by harshilparmar**
@aamay001 Brother,I have made pr for #72 work section.As we discussed I have changed code.Please check and let me know if it needs any other changes 👍

### CODE REVIEW

File: src/components/Resume/Experience.js [hunk 1]
```diff
@@ -46,8 +42,8 @@ Experience.propTypes = {
   font: PropTypes.string.isRequired,
 };
 
-const mapStateToProps = state => ({
-  experience: state.resume.experience,
+const mapStateToProps = (state) => ({
+  experience: migrateSchema(state.resume).work,
```
Comment (aamay001) [inline src/components/Resume/Experience.js:46]: This should not be here. There is no need to include the `migrateSchema` function. Please revert this line. The `migrateSchema` method will be called outside from here.

**Review by aamay001 — CHANGES_REQUESTED** [review]
I added some additional instructions to the issue #72 

Please update `migratedresume-data.js` by removing the `experience` property and replacing it with the `work` property from the new schema.

**Comment by aamay001** [comment]
FYI, the `CodeEditor` should show your changes and should be able to update the resume using the new schema for the work section.

**Comment by aamay001** [comment]
Please submit a new PR when changes are completed.

File: src/components/Resume/Experience.js (hunk 1 above)
Comment (harshilparmar) [reply src/components/Resume/Experience.js:46]: Sorry, If my code caused you trouble.I will change it.

**Pull request closed without merging**